web/app/themes/justicejobs/src/js/accessible-carousel.js

```diff
@@ -78,83 +78,86 @@ jQuery(document).ready(function ($) {
 
       // Select the element and the individual slides
       carousel = document.getElementById(settings.id);
-      slides = carousel.querySelectorAll('.accessible-carousel__slide');
 
-      carousel.className = 'carousel';
+      if (carousel) {
```

Comment:
Hey, if the variable becomes undefined, JavaScript will throw an error. Best prcatice (in js) is to always check for null.

Reply:
This was to fix an error and didn't throw any errors that I could find across Chrome or Firefox. I've slightly refactored it though.